### ISSUE TITLE
Bluesky ignores RTA if the drone has to fly really slow

### ISSUE THREAD

**Issue by undefiened** [TUDelft-CNS-ATM/bluesky#310]
Here is an example of a scenario that causes an issue (I simplified it as best as I can to isolate the issue).
Bluesky completely ignores RTA on all of the waypoints. It just slows down before any of the waypoints and continues to fly.

```
00:00:00>HOLD
00:00:00>ASAS ON
00:00:00>PAN 48.223775 16.337976
00:00:00>ZOOM 50
00:00:00>CRE D0 M600 48.2173376 16.3416682 15.372171339863279 0.0 5
00:00:00>DEFWPT D0_0,48.2173376,16.3416682,FIX
00:00:00>DEFWPT D0_1,48.21788434433063,16.3418931204952,FIX
00:00:00>DEFWPT D0_2,48.2184919,16.3421405,FIX
00:00:00>DEFWPT D0_3,48.21915898133572,16.342406596925375,FIX
00:00:00>ADDWPT D0 D0_1 0.0 ,
00:00:00>ADDWPT D0 D0_2 0.0 ,
00:00:00>RTA D0 D0_2 0:01:09
00:00:00>ADDWPT D0 D0_3 0.0 ,
00:00:00>RTA D0 D0_3 0:01:15
00:00:00>LNAV D0 ON
00:00:00>VNAV D0 ON
```

**Comment by jooste** [comment]
Thanks for raising this issue. It seems that after the first waypoint, the aircraft does the opposite (speeding up) of what is required (slowing down). We'll have a look!

**Comment by jooste** [comment]
The problem was that commanded speeds lower than 2 m/s are were interpreted as Mach numbers. This is resolved in 6a5e139 with the addition of a new stack command: `CASMACHTHR`, that you can use to set this threshold. If you set it to zero no CAS/Mach checking will be done, which is the desired behaviour for drones.
